app/templates/sectionsummary.html

```diff
@@ -0,0 +1,37 @@
+{% import 'macros/helpers.html' as helpers %}
+
+{% extends theme('section_confirmation.html') -%}
+
+{% block page_title -%}{{_("Section-Summary")}} - {{survey_title}}{% endblock -%}
+
+{% block sidebar %}
+  {% if navigation %}
+    {% include theme('partials/navigation.html') %}
+  {% endif %}
+{% endblock %}
+
+{% block form_content %}
+{% endblock %}
+
+{% block main -%}
+
+<div>
+    {% set group = content.summary.groups | first %}
+        <h1 class="summary__title saturn" id="{{group.id}}">{{group.title}}</h1>
```

Comment:
The indentation is incorrect here.
also, what happens if the group title is blank?
should this not be the section title?